clippy_lints/src/disallowed_name.rs

```diff
@@ -17,33 +17,33 @@ declare_clippy_lint! {
     /// ```rust
     /// let foo = 3.14;
     /// ```
-    pub BLACKLISTED_NAME,
+    pub DISALLOWED_NAME,
```

Comment:
Let's rename it properly:

```suggestion
    pub DISALLOWED_NAMES,
```

Reply:
I believe it's the lint definition which is in singular:
```rust
#![warn(clippy::blacklisted_name)]
```
not the toml entry:
```toml
blacklisted-names = ["toto", "tata", "titi"]
```
which is in plural.

Reply:
Yeah, but it should be in plural, as per our [lint naming conventions](https://rust-lang.github.io/rfcs/0344-conventions-galore.html#lints) (point 3)